tests/test_main.py

```diff
@@ -0,0 +1,79 @@
+import asyncio
+import unittest
+from unittest import mock
+
+import asyncpg as asp
+import pytest
+
+from autoagora.config import args
+from autoagora.main import (
+    DEFAULT_AGORA_VARIABLES,
+    allocated_subgraph_watcher,
+    asyncpg,
+    init_config,
+    metrics_server,
+)
+
+
+class TestScript:
+    @pytest.fixture
+    def pgsql(self, postgresql):
+        return postgresql
+
+    async def test_allocated_subgraph_watcher(self, pgsql):
+        subgraph1 = "QmPnu3R7Fm4RmBF21aCYUohDmWbKd3VMXo64ACiRtwUQrn"
+        subgraph2 = "QmTJBvvpknMow6n4YU8R9Swna6N8mHK8N2WufetysBiyuL"
+        with mock.patch("autoagora.main.init_config") as mock_init_config:
+            # with mock.patch("autoagora.main.asyncpg") as mock_asyncpg:
+            with mock.patch(
+                "autoagora.main.get_allocated_subgraphs"
+            ) as mock_get_allocated_subgraphs:
+                with mock.patch("autoagora.main.set_cost_model") as mock_set_cost_model:
+                    with mock.patch(
+                        "autoagora.main.model_update_loop"
+                    ) as mock_model_update_loop:
+                        with mock.patch(
+                            "autoagora.main.price_bandit_loop"
+                        ) as mock_price_bandit_loop:
+                            mock_init_config.return_value = None
```

Comment:
Doesn't it already always return `None`?

Reply:
oh yeah, you are right that was from some test  was doing, this one needs some clean up